cookbooks/fb_apache/attributes/default.rb

```diff
@@ -18,6 +33,15 @@
 auth_core_suffix = node.centos6? ? 'default' : 'core'
 
 default['fb_apache'] = {
+  'default_sites_enabled' => true,
+  'default_sites' => [
```

Comment:
I don't think we should do this at all. The whole point of the model we have is all the sites are in one file managed by the cookbook. We should wholesale remove everything from sites-enabled other than our file.